kube/client.go

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package kube provides a client to interact with Kubernetes.
```

Comment:
Bit fat warning that this is Tailscale-internal and not meant for external consumption and is not a stable API.